worker-framework/docs/pages/en-us/Features.md

```diff
@@ -8,7 +8,7 @@ banner:
     subtitle: Analyze a Larger Range of Formats
```

Comment:
I was going to do this once the review had been completed. Does that sound okay?